source/common/http/http1/codec_impl.cc

```diff
@@ -497,10 +508,17 @@ Status ConnectionImpl::completeLastHeader() {
   ASSERT(dispatching_);
   ENVOY_CONN_LOG(trace, "completed header: key={} value={}", connection_,
                  current_header_field_.getStringView(), current_header_value_.getStringView());
+  auto& headers_or_trailers = headersOrTrailers();
+  StreamInfo::BytesMeterer* bytes_meterer = getBytesMeterer();
+  if (bytes_meterer) {
+    // Account for ":" bytes between the header key value pair.
+    bytes_meterer->addHeaderBytesReceived(headers_or_trailers.size());
+    // Account for "\r\n" at the end of headers.
+    bytes_meterer->addHeaderBytesReceived(CRLF_SIZE * headers_or_trailers.size());
```

Comment:
I'm fairly sure this is a bug since `completeLastHeader()` gets called every time we complete a header field:value pair -- this is NOT necessarily "we're done with all headers". Hence our counting can be drastically off if we have tiny headers.

Just want to add 3 bytes here instead of `header_or_trailers.size()` and  `2 * header_or_trailers.size()`